src/Screens/Configuration/ChangeData/ChangeDataScreen/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Container, Text } from 'native-base';
+import React from 'react';
+
+import Styles from './styles';
+
+const ChangeData: React.FunctionComponent = () => {
```

Comment:
Maybe we should create the screens structure in a different branch first so it focus only in one task

Reply:
I agree, but since we had already started the task, Seabra and I did it together, I committed and then he merged this commit into his branch, I don't know if it will be a problem. But maybe we can leave this way, just for the sake of ease kkk if you agree.